content/communityExtensions/figmaToBabylonJS.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Figma to BabylonJS
+image: /img/extensions/FigmaToBabylonJS.png
+description: FigmaToBabylonJS is a Figma plugin which allows you to export  Figma designs to BabylonJS GUI .json files.
+keywords: extensions, Figma, GUI, BabylonGUI, Design, UI, plugin
+further-reading:
+video-overview:
+video-content:
+---
+
+# Figma to BabylonJS
+
+## Introduction
+
+![Screenshot showing the figma right click menu on the Plugins option](/img/extensions/FigmaToBabylonJS.png)
+
+FigmaToBabylonJS is a Figma plugin which allows you to export Figma designs to BabylonJS GUI .json files. These can be parsed into a fullscreen UI [Advanced Dynamic Texture](/features/featuresDeepDive/gui/gui#advanceddynamictexture).
+
+
+## Get the Plugin
+
+The plugin is available via the [Figma community](https://www.figma.com/community/plugin/1186201881571137432) page.
+
+## How to Use
+
+After adding the plugin to Figma, you need to highlight the frame/artboard that you want to export and right click on it.
+On the rightclick menu, select Plugins > Saved Plugins > FigmaToBabylonJS
+![Screenshot showing the figma right click menu on the Plugins option](/img/extensions/FigmaToBabylonJS-Step1.png)
+
+Choose if you want the output to include the background color (changes the "root" background color to the frames fill color)
+Click the “Convert to BabylonJS JSON” button and after a short wait you will see your file explorer. Choose where you want the file to go. For best results, place it directly into your project with the correct file name your code uses, this way the UI will automatically update.
+![Screenshot showing the figma right click menu on the Plugins option](/img/extensions/FigmaToBabylonJS-Step2.png)
+
+It's also good practice to load the UI in gui.babylonjs.com or in your project but loading in the inspector so you can edit the UI after the fact. Just to make sure that the exporting has went the way you expected and there weren't any strange inconsistencies with the design [see inconsistencies and polyfills](/communityExtensions/figmaToBabylonJS#inconstancies--polyfills)
```

Comment:
"It's also good practice to load the UI in gui.babylonjs.com or in your project but loading in the inspector so you can edit the UI after the fact" 
This sentence is a little confusing. You are saying it's good practice to load the UI into gui.babylonjs.com, but the second part of the sentence is not clear "or in your project but loading in the inspector".

I think you are saying you "should preview the UI by loading it into either gui.babylonjs.com or your local project using the inspector" but the original wording is a bit confusing

Reply:
Agreed, fixed in my latest commit.